proximal/examples/test_deconv_sinv_psf.py

```diff
@@ -49,18 +54,22 @@ def adjoint(im_src, im_sim):
     
     return LinOpFactory(im_shape, im_shape, forward, adjoint)
 
-def cost_function(x, im, psf, mu=0.1): 
-    blur_op = my_factory(psf, im.shape)
-    im_blur = blur_op(x)
+def cost_function(x, im, psf, mu=0.1, using_px_conv=True): 
+    if using_px_conv:
+        im_blur = px.conv(psf2_cropped, x)
+    else:
+        blur_op = my_factory(psf, im.shape)
+        im_blur = blur_op(x)
+
     data_term = px.sum_squares(im_blur - im) 
-    im_grad = px.grad(im_blur)
-    grad_term = mu * px.norm1(im_grad) #+ (1 - mu) * px.sum_squares(im_grad)
+    im_grad = px.grad(x)
+    grad_term = mu * px.group_norm1(im_grad, group_dims=(2, )) #+ (1 - mu) * px.sum_squares(im_grad)
```

Comment:
Image deconvolution, an inverse problem, needs to be regularized (aka numerically stabilized) by `grad(x)` to be effective. Otherwise, ProxImaL can be tricked to optimize for `im_blur`, or `x`, hence defeating the original intent of deconvolution. 

Please feel free to change it back if you disagree.

Minor edit: `norm1` to `group_norm1`, i.e. isotropic TV, in order to inform ProxImaL the presence of slanted edges in the test image.

Reply:
Closing this thread because @shnaqvi intended to use anisotropic TV as the prior.